### PR TITLE
Migrationify

### PR DESCRIPTION
**Pull request by rashmirathi1729**

### CODE REVIEW

File: spec/author_spec.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  before do
+    blog = Blog.create!(name: 'A Blog')
+    @author = blog.authors.create!(name: 'Ben', twitter_handle: '@benlovell')
+  end
+
+  it 'has a twitter handle' do
+    @author.twitter_handle.must_equal('@benlovell')
+  end
+
+  it 'is valid with valid attributes'  do
```
Comment (benlovell) [inline spec/author_spec.rb]: Whoops, just one extra space before the `do` here.

File: spec/blog_spec.rb [hunk 2]
```diff
@@ -1,22 +1,52 @@
+require 'pry'
```
Comment (benlovell) [inline spec/blog_spec.rb]: Perhaps you might consider requiring pry from the helper?

File: spec/post_spec.rb [hunk 3]
```diff
@@ -1,13 +1,51 @@
+require 'pry'
```
Comment (benlovell) [inline spec/post_spec.rb]: Same here.

**Comment by benlovell** [comment]
You've committed the `db/blog_development.sqlite3-journal` which you might want to remove now?

**Comment by benlovell** [comment]
This is pretty much perfect, just those very very minor comments to address but I'd be entirely happy to merge this regardless. Extra props for running rubocop! :heart_eyes:

**Comment by rashmirathi1729** [comment]
@benlovell Removied 'require pry' from test files and also removed the database files as you asked.

**Comment by rashmirathi1729** [comment]
@benlovell Moved 'require pry' to helper file and also removed the database files.

**Comment by benlovell** [comment]
:+1: nice job! All good :heart:

**Pull request closed without merging**